rust/examples/otel-stdout.rs

```diff
@@ -39,11 +40,12 @@ pub async fn main() -> anyhow::Result<()> {
         .get_func(&mut store, function_name)
         .expect("function exists");
 
-    trace_ctx.set_trace_id("any-old-trace-id".to_string()).await;
+    trace_ctx.set_trace_id(new_trace_id()).await;
```

Comment:
This means that the user can't necessarily set any string id in this adapter. i think that's okay and we can look into it in the future. I'm not sure it's a safe assumption that it can be supported everywhere. a random u64 is the safest option in terms of compatibility and conflict.